#227 project structure

robinweser opened the pull request:
closes #227

Commit:
#227 updating config files

Commit:
renamed build to scripts

Commit:
#227 moved demo to packages

Commit:
#227 moved resources to root folder

Commit:
#227 moved dom docs to packages

Commit:
#227 moved all dom modules and utils to packages

Commit:
#227 moved dom tests to packages

Commit:
#227 moved shared files to comon

Commit:
#227 added global testing scripts

Commit:
fixed testcase

Commit:
#227 moved native docs to packages

Commit:
#227 moved native modules to packages

Commit:
#227 added test files and utils for native tests

Commit:
running tests on multiple node versions

Commit:
added missing dependecies

Commit:
#227 added lodash dependency for build

Commit:
updated main README

Commit:
added esformatter packages

alitaheri commented:
@rofrischmann Wow, you're working hard on this :heart_eyes: :heart_eyes: Thanks a lot :+1:

robinweser commented:
@alitaheri Hehe yeah, that was the first iteration. Now I'll fix the tests and clean em up. Afterwards add missing tests to push coverage again (by now dropped to 76 only as I removed legacy stuff)
